vmp/helper/tf.go

```diff
@@ -0,0 +1,36 @@
+package helper
+
+// InterfaceToStringArray converts a interface{} whose underlying type is []string. Useful for parsing Terraform resource values.
+func InterfaceToStringArray(attr interface{}) []string {
+	if attr == nil {
+		return nil
+	}
+
+	// Terraform's schema.TypeList stores values as []interface{}
+	if interfaceArray, ok := attr.([]interface{}); ok {
```

Comment:
We need to make sure that underlying type of attr is array of interface before type conversion.